Exercises/4-difference.js

```diff
@@ -1,8 +1,15 @@
 'use strict';
 
-// Find difference of two arrays
-// elements from array1 that are not includes in array2
 
-const difference = (array1, array2) => [];
+const difference = (array1, array2) => {
+  const res = [];
+  for (const el of array1) {
+    if (!array2.includes(el)) res.push(el);
+  }
+  return res;
+};
 
 module.exports = { difference };
+
+
+
```

Comment:
Don't leave empty lines at the end of file, by default we must leave single empty line a EOF